x-pack/plugins/upgrade_assistant/common/constants.ts

```diff
@@ -41,3 +41,23 @@ export const CLUSTER_UPGRADE_STATUS_POLL_INTERVAL_MS = 45000;
 export const CLOUD_BACKUP_STATUS_POLL_INTERVAL_MS = 60000;
 export const DEPRECATION_LOGS_COUNT_POLL_INTERVAL_MS = 15000;
 export const SYSTEM_INDICES_MIGRATION_POLL_INTERVAL_MS = 15000;
+
+/**
+ * List of Elastic apps that potentially can generate deprecation logs.
+ * We want to filter those out for our users so they only see deprecation logs
+ * that _they_ are generating.
+ */
+export const APPS_WITH_DEPRECATION_LOGS = [
+  'kibana',
```

Comment:
Will this exclude any 3rd-party plugins that might be throwing deprecation logs? Should we consider them anyhow?

Reply:
No, for now we only exclude Elastic products and we haven't talked about third party plugins. We _know_ that for our product we have handled the deprecation in `8.0`, we don't have that guarantee with external plugins.